src/_codelabs/index.md

```diff
@@ -15,19 +15,15 @@ See the latest Dart codelabs on the Google Developers site:
 * [Building Beautiful UIs with Flutter](https://codelabs.developers.google.com/codelabs/flutter)
 * [Firebase for Flutter](https://codelabs.developers.google.com/codelabs/flutter-firebase)
 
-**AngularDart**
-* [Write a Material Design AngularDart Web App](https://codelabs.developers.google.com/codelabs/your-first-angulardart-web-app)
-* [Build an AngularDart & Firebase Web App](https://codelabs.developers.google.com/codelabs/angulardart-firebase-web-app)
-
-## Other codelabs
-
-The following links point you to codelabs for Dart technologies:
-
-* [Dart for the Web codelabs](https://webdev.dartlang.org/codelabs)
-* Search for "Flutter" on the
-  [Google Developer codelabs site](https://codelabs.developers.google.com/)
-
 {% comment %}
 Seth hasn't yet added links to these codelabs on flutter.io. If and
 when he does, link to that.
 {% endcomment %}
+
+**AngularDart**
+* [Write a Material Design AngularDart Web App](https://codelabs.developers.google.com/codelabs/your-first-angulardart-web-app)
+* [Build an AngularDart & Firebase Web App](https://codelabs.developers.google.com/codelabs/angulardart-firebase-web-app)
+
+For a full list of Dart web codelabs, see
+[Codelabs](https://webdev.dartlang.org/codelabs) on
+[Dart for the web](https://webdev.dartlang.org).
```

Comment:
I'd personally just say "webdev.dartlang.org" instead of "Dart for the web" (which could change).

We could even just give them the URL in plain text (less repetition that way):

For a full list of Dart web codelabs, see [webdev.dartlang.org/codelabs](...).